### DISCUSSION TITLE
Trying to refine search results by only results in which a column has data

### DISCUSSION THREAD

**Discussion by vickyhuet** [microsoft-search/pnp-modern-search#3611] [Q&A] [answered]
I am using PnP v4. I simply want the results to display only when RefinableString123 contains data. How do I do this? I tried RefinableString123:"" and RefinableString123:0

**Comment by kasperbolarsen** [comment, marked as answer]
Refinablestring123:* should do the trick :-)

**Reply by vickyhuet** [reply]
@kasperbolarsen thank you so much! That worked and was so simple :) Have a great day!

**Reply by wobba** [reply]
Also in the KQL docs https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference#filter-on-items-where-a-text-property-is-empty-or-contains-a-value
